src/Esfa.Vacancy.Api.Types/SearchApprenticeshipParameters.cs

```diff
@@ -12,6 +12,12 @@ public class SearchApprenticeshipParameters
         /// </summary>
         public string FrameworkLarsCodes { get; set; }
 
+        /// <summary>
+        /// The training provider's UKPRN.
```

Comment:
Just a naming thing... I think we should just use UKPRN as the abbreviation is specifically for provider.

Reply:
I did this originally but in the returned document we already have `TrainingProviderName`. So I added `TrainingProviderUkprn` too. So Then I though I should call in parameter the same thing so its the same.

Reply:
Perhaps, the model's which are exposed to the public can be use Ukprn, as it reads wrong if you expand the abbreviation. 

For instance:
> The training provider's UKPRN.

This will read **The training provider's unique provider reference number**

Reply:
👍